project-base/src/Shopsys/ShopBundle/Controller/Front/ProductController.php

```diff
@@ -175,13 +174,7 @@ public function listByCategoryAction(Request $request, $id)
             'priceRange' => $productFilterConfig->getPriceRange(),
         ];
 
-        if ($request->isXmlHttpRequest()) {
-            return $this->render('@ShopsysShop/Front/Content/Product/ajaxList.html.twig', $viewParameters);
-        } else {
-            $viewParameters['POSITION_PRODUCT_LIST'] = Advert::POSITION_PRODUCT_LIST;
-
-            return $this->render('@ShopsysShop/Front/Content/Product/list.html.twig', $viewParameters);
-        }
+        return $this->render('@ShopsysShop/Front/Content/Product/ajaxList.html.twig', $viewParameters);
```

Comment:
Please, can you explain the thought behind this change? I'm a bit baffled.

Reply:
Oh, it was by mistake, thanks for noticing that :+1: I though that the template was the same but now I see it was not :man_facepalming:

Reply:
fixed :innocent: